lib/components/T3Renderer/T3Renderer.js

```diff
@@ -44,8 +50,8 @@ export default {
     // render loop of content elements
     return ctx.props.content.map((element, index) =>
       ctx.props.frame && element.appearance.frameClass !== 'none'
-        ? renderFrame(element, index)
-        : renderComponent(element, index)
+        ? renderFrame(element, index, `c${element.id}`)
```

Comment:
i think we can avoid third parameter here (for frame and component), because we already passed `element` here - what do you think ?